src/thunderbots/software/ai/intent/all_intents.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
```

Comment:
OK, so:
- we don't actually include all the headers in the `IntentVisitor`, we just pre-declare them all. This is because if we include them there is a circular dependency between the `IntentVisitor` and every Intent
- as such, everything that extends `IntentVisitor` needs to include all the intents. Since there are several classes that do, I thought it was easier to have one header to include all the intents.
- I don't think it will be too much of an issue to remember because the compiler will explicitly tell you something is wrong?

Definitely open to other suggestions here though!

Reply:
Hmmmm ok let's see how it goes for now then.